src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1497,7 +1497,7 @@ If such a class is used as a base class and if the deriving class defines a dest
     <value>'{0}' in explicit interface declaration is not an interface</value>
   </data>
   <data name="ERR_InterfaceMemberNotFound" xml:space="preserve">
-    <value>'{0}' in explicit interface declaration is not a member of interface</value>
+    <value>'{0}' in explicit interface declaration is not found among members of the interface that can be implemented</value>
   </data>
```

Comment:
> This feels like a regression in the quality of the diagnostic. 

I don't see this as a regression. In fact I believe the change in the error message wording is an improvement and the message now more accurately describes the cause. I agree that it is not as detailed as it could be, but it wasn't that detailed before regarding the static members, the same handling is simply expanded to all non-virtuals. Accessibility plays no role for this diagnostics at the moment. I made a note in https://github.com/dotnet/roslyn/issues/17952 about possibly providing more details in the message.